cairo_programs/proof_programs/usort.cairo

```diff
@@ -0,0 +1,22 @@
+%builtins range_check
```

Comment:
Move this file to be a symlink to the one under the `cairo_programs` directory

Reply:
Done [fe15372](https://github.com/lambdaclass/cairo-vm_in_go/pull/244/commits/fe1537281335b8887bd6dd42d6cb1824f7b63c41)